examples/agents/in-house-ctf/README.md

```diff
@@ -0,0 +1,57 @@
+# In House CTF Tasks
+This directory includes Capture The Flag challenges migrated from Google DeepMind's [Dangerous Capabilities Evaluations](https://github.com/google-deepmind/dangerous-capability-evaluations/tree/main/in_house_ctf) (originally published in [Evaluating Frontier Models for Dangerous Capabilities](https://arxiv.org/pdf/2403.13793)). Here we include 9 of the 13 In-House CTF challenges (excluding 4 challenges that have faulty Docker images from the original repository). 
+
+This implementation uses [tool calling](https://inspect.ai-safety-institute.org.uk/tools.html) to enable the model to execute bash commands. The flag values has been replaced with placeholders `REPLACE_ME_WITH_RANDOM_FLAG_N` to prevent them from ending up in scrapable GitHub data, the actual values will be updated during evaluation run.
+
+Note that the bash commands are executed inside Docker containers, so you will need to install [Docker Engine](https://docs.docker.com/engine/install/) in order to run the evaluation. Each task runs an attacker container and one or more challenge enviroment containers using the task-specific `compose.yaml` files, and the Dockerfiles for these tasks
+are adapted from the original repository with minor modifications. 
+
+To evaluate against various models:
```

Comment:
It looks like the compose files are pointing to pre-built images -- should we note that?

Reply:
If that's true, we should also note that the Dockerfiles are included for documentation / if people want to build their own images but aren't actually used by the compose files.

Reply:
Will make some updates to further clarify this